libs/portals/admin/application-system/src/components/ApplicationsTable/ApplicationsTable.tsx

```
@@ -88,9 +90,15 @@ export const ApplicationsTable = ({
         <T.Head>
           <T.Row>
             <T.HeadData>{formatMessage(m.dateCreated)}</T.HeadData>
-            <T.HeadData>{formatMessage(m.application)}</T.HeadData>
+            {!showAdminData && (
+              <T.HeadData>{formatMessage(m.application)}</T.HeadData>
+            )}
             <T.HeadData>{formatMessage(m.applicant)}</T.HeadData>
             <T.HeadData>{formatMessage(m.nationalId)}</T.HeadData>
+            {showAdminData &&
+              applications[0]?.adminData?.map((x) => (
+                <T.HeadData>{formatMessage(x.label) ?? ''}</T.HeadData>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)